cms/djangoapps/contentstore/views/videos.py

```diff
@@ -412,7 +412,7 @@ def make_csv_dict(video):
             ]
         )
         return {
-            key.encode("utf-8"): value.encode("utf-8")
+            key.encode("utf-8") if six.PY2 else key: value.encode("utf-8") if six.PY2 else value
```

Comment:
These functions should ideally always return bytes on unicode, this seems like it will do one thing in python 2 and another in python 3.

Reply:
in python 3 if we encode the `key` and `value` here it converts them to `bytes` and it fails in `csv` module because `csv` module expects `str` type `key` and `value` of the given dictionary while writing the `csv`